data-access/basic_data_access.ipynb

```diff
@@ -167,538 +248,1117 @@
+    "- `export` - Exports your requested dataset to a range of file formats.\n",
     "\n",
-    "### Location subsetting \n",
-    "The options for location subsetting can be found using the `get_data_options()` function, as described in the beginning of this notebook. You can also opt to perform an area average by setting `area_average = \"Yes\"`. The `get_data()` function will default to returning the entire spatial domain, with no area averaging performed. \n",
-    "- area_subset (optional) \n",
-    "- cached_area (optional) \n",
-    "- area_average (optional)\n",
+    "The first three processors (bolded) are run by default every time that you retrieve data. Examples of other available processors can be found in the `climakitae` library documentation, or in other example notebooks. <br><br>\n",
+    "It's important to note that processors are applied as a **dictionary**. This enables you to add more than one processor to your chain of operations. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96754193-6c30-4ece-b712-f3082b97cdd9",
+   "metadata": {},
+   "source": [
+    "## Processor Examples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ca5c026-c842-4fa4-ab92-6d72069d10be",
+   "metadata": {},
+   "source": [
+    "To bring cohension between the processor examples we'll be looking at below, we'll be building towards a complete, realistic analysis using WRF data. Any dataset works, but for demonstration purposes it is useful to see how the puzzle pieces fit together using a consistent dataset."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "654ace5a",
+   "metadata": {},
+   "source": [
+    "### Processor Example 1: Concatenation along a specified dimension\n",
+    "By default, when historical data is retrieved in the same operation as future data, the historical data will be appended to the future data, giving a single timeseries. However, you can change this default behavior by setting the query to concatenate along the simulation `sim` dimension instead. This will return the historical and future data as separate simulations. Concatenating by `time` or by `sim` have unique benefits, and you'll need to decide which method is most appropriate for your analyses. \n",
     "\n",
-    "### Additional options\n",
-    "Further modify the data returned using the following arguments.\n",
-    "- downscaling method (optional)\n",
-    "- approach (optional) \n",
-    "- scenario (optional)\n",
-    "- units (optional)\n",
-    "- time_slice (optional)\n",
-    "- warming_level (optional)\n",
-    "- warming_level_window (optional)\n",
-    "- warming_level_months (optional)"
+    "In the returned data from the code below, future time periods for the historical simulation will be infilled with `NaN`, because the model has no data for that time period by definition. The same logic applies to the future simulations: any time in the past will be infilled with `NaN`. "
```

Comment:
Would it be possible to be more specific about which periods are filled with NaN? For example 'the returned historical data will contain only NaN values for years after 2015 (or whatever the year should be), as that is the final year of the model's 'historical' time period.'